### PR TITLE
Docs: Added a How-to guide to use API key

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Added documentation on how-to use API key.
Added image from Postman.

### CODE REVIEW

File: content/how-to-guides/how-to-use-api-key.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+Title: How to use API key
+---
+
+# Steps to use API Key
+
+This guide helps you with the steps to use the API key to interact with the Stacks Blockchain API.
+
+1. [Send a request](https://survey.hiro.so/hiroapi) to the Hiro team to get an API key. You will get the response in the form of the following:
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply content/how-to-guides/how-to-use-api-key.md:9]: @sabbyanandan Could you help with the correct link to the request form?

File: content/how-to-guides/how-to-use-api-key.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Using Curl
+
+The following section walks you through using an API key with Curl.
+
+`curl https://api.hiro.so -H 'x-hiro-api-key: <API_KEY>'`
+
+Now, put the API key value that you received from your request, and the command now looks like this: 
+
+`curl https://api.hiro.so/extended/v1/microblock -H 'x-hiro-api-key: Nabcs1234efg56789aaaaaaaaqqqqqqqeeeeee12334345'`
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply content/how-to-guides/how-to-use-api-key.md]: Done!

[the remark this answers is not in the document]
Comment (CharlieC3) [reply content/how-to-guides/how-to-use-api-key.md]: Perfect! Thanks!

File: content/how-to-guides/how-to-use-api-key.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sabbyanandan) [reply content/how-to-guides/how-to-use-api-key.md:9]: Hi. This form is to gather interest. I am confused why we are adding it here as part of "[How-to guide to use API key]" guide. We don't need it here.

This guide should be about how a dev will use an API to make the HTTP requests, which is captured in the CURL command below. 

Also show another example of how to do it in a Typescript function for additional guidance.

File: content/how-to-guides/how-to-use-api-key.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+---
+Title: How to use API key
+---
+
+API keys are alpha-numeric string values distributed to the developer customers to grant access to your API.
```
Comment (sabbyanandan) [inline content/how-to-guides/how-to-use-api-key.md:5]:
```suggestion
API keys are alpha-numeric codes that identify and authenticate an application or developer.
```

File: content/how-to-guides/how-to-use-api-key.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+# Steps to use API Key
+
+This guide helps you with the steps to use the API key to interact with the Stacks Blockchain API.
```
Comment (sabbyanandan) [inline content/how-to-guides/how-to-use-api-key.md:9]: As discussed, let's generalize this document in its entirety, given that this guide applies to all 3 APIs we ship/support. 

We may have to find a way to reuse this guide for all 3 product sections, also.

File: content/how-to-guides/how-to-use-api-key.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+:::note
+
+_The API key is passed in the **header** of your API call._
+
+:::
+
+There are multiple ways to interact with the endpoint. However, this document walks you through three ways to make an [API call](https://docs.hiro.so/api#tag/Microblocks/operation/get_microblock_list). You can choose between any of the following methods.
```
Comment (sabbyanandan) [inline content/how-to-guides/how-to-use-api-key.md:17]:
```suggestion
There are multiple ways to interact with the API endpoints. However, this document walks you through three ways to make an [API call](https://docs.hiro.so/api#tag/Microblocks/operation/get_microblock_list). You can choose between any of the following methods.
```

File: content/how-to-guides/how-to-use-api-key.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+The following section walks you through using an API key with [Postman](https://www.postman.com/).
+
+In the Postman request, add the [API endpoint](https://api.hiro.so/extended/v1/microblock) with the `Get` request. Then, add the API key and its value received from the Hiro team in the **Headers** section, as shown in the image below.
+
+Select **Send** to get recent microblocks.
+
+![API-Key](../images/api-key.jpeg)
+
+## Using the Typescript function
```
Comment (sabbyanandan) [inline content/how-to-guides/how-to-use-api-key.md:39]:
```suggestion
## Using the API key in Typescript
```

Comment (sabbyanandan) [reply content/how-to-guides/how-to-use-api-key.md:39]: @rafaelcr: You may have some commentary on the TS function. Feel free to suggest-edit.

Comment (rafaelcr) [reply content/how-to-guides/how-to-use-api-key.md:39]: Code sample LGTM

**Comment by LakshmiLavanyaKasturi** [comment]
Discarding the pull request

**Comment by LakshmiLavanyaKasturi** [comment]
The pull request replacing this is https://github.com/hirosystems/docs/pull/381

**Pull request closed without merging**